README.md

```diff
@@ -79,14 +51,54 @@ Requires `rust` and `cargo`: [installation instructions.](https://www.rust-lang.
 $ cargo build [--release]
 ```
 
+### Test
+
+```
+$ cargo test --release
+```
+
 ### Example Network Simulation
 
 A basic [example](https://github.com/poanetwork/hbbft/blob/master/examples/README.md) is included to run a network simulation.
 
 ```
-$ cargo run --example simulation -- -h
+$ cargo run --example simulation --release -- -h
 ```
 
+## Current TODOs
+
+See [Issues](https://github.com/poanetwork/hbbft/issues) for all tasks in progress. 
+
+- [ ] Honey Badger encryption ([#41](https://github.com/poanetwork/hbbft/issues/41))
```

Comment:
It's almost not worth adding that anymore. #68 is about to be merged today or tomorrow.